app/serializers/museum_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class MuseumSerializer < ActiveModel::Serializer
+  # attributes :id
+end
```

Comment:
I would remove this serializer if you're not using it, as it will keep ANYTHING from showing for a museum!